iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -88,21 +87,15 @@
 
 NSString* const kOSSettingsKeyAutoPrompt = @"kOSSettingsKeyAutoPrompt";
 
-/* Enable the default in-app alerts*/
-NSString* const kOSSettingsKeyInAppAlerts = @"kOSSettingsKeyInAppAlerts";
-
 /* Enable the default in-app launch urls*/
 NSString* const kOSSettingsKeyInAppLaunchURL = @"kOSSettingsKeyInAppLaunchURL";
 
-/* Set InFocusDisplayOption value must be an OSNotificationDisplayType enum*/
-NSString* const kOSSettingsKeyInFocusDisplayOption = @"kOSSettingsKeyInFocusDisplayOption";
```

Comment:
We can remove this now as well since `OSNotificationDisplayType` is decided from the `NotificationWillShowInForegroundHandler`

Reply:
removed!